streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignorTest.java

```diff
@@ -131,7 +131,7 @@ public void testSubscription() throws Exception {
         String clientId = "client-id";
         UUID processId = UUID.randomUUID();
         StreamThread thread = new StreamThread(builder, config, new MockClientSupplier(), "test", clientId, processId, new Metrics(), Time.SYSTEM, new StreamsMetadataState(builder, StreamsMetadataState.UNKNOWN_HOST),
-                                               0) {
```

Comment:
All the changes on this file seems to be coding style only, are they intentional?